app/services/core/rx-entity-proxy.ts

```diff
@@ -64,7 +65,13 @@ export abstract class RxEntityProxy<TParams, TEntity> extends RxProxyBase<TParam
                 if (error.status === HttpCode.NotFound) {
                     this._itemKey.next(null);
                     const queryKey = this.params[this.cache.uniqueField];
-                    this.cache.deleteItemByKey(queryKey);
+                    if (queryKey) {
+                        this.cache.deleteItemByKey(queryKey);
+                    } else {
+                        // tslint:disable:max-line-length
```

Comment:
oh yeah, of course. i grabbed the first one i saw, the next line was the one i wanted.